R/pkg/R/mllib.R

```diff
@@ -51,13 +45,12 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' summary(model)
 #'}
 setMethod("glm", signature(formula = "formula", family = "ANY", data = "DataFrame"),
-          function(formula, family = c("gaussian", "binomial"), data, lambda = 0, alpha = 0,
-            standardize = TRUE, solver = "auto") {
+          function(formula, family = c("gaussian", "binomial", "poisson", "gamma"), data,
```

Comment:
We should match R's signature for `family` now. Internally, we can support family/link functions R supports. If user input `binomial("logit")`, we can get the family name and the link name before we call the Scala implementation.